src/helpers/check-merge-safety.ts

```diff
@@ -96,12 +100,21 @@ const getMergeSafetyMessage = async (
     basehead: `${baseOwner}:${default_branch}...${username}:${ref}`
   });
   const changedFileNames = changedFiles?.map(file => file.filename);
-  const projectDirectories = paths?.split(/[\n,]/);
-  const isUnsafeToMerge = projectDirectories?.some(
+  const changedProjectDirectories = paths?.split(/[\n,]/);
+
+  const changedProjectsOutdatedOnBranch = changedProjectDirectories?.filter(
     dir => fileNamesWhichBranchIsBehindOn.some(file => file.includes(dir)) && changedFileNames?.some(file => file.includes(dir))
   );
 
-  if (isUnsafeToMerge) {
+  if (changedProjectsOutdatedOnBranch?.length) {
+    core.error(buildErrorMessage(changedProjectsOutdatedOnBranch));
     return `This branch has one or more outdated projects. Please update with ${default_branch}.`;
   }
 };
+
+const buildErrorMessage = (paths: string[]) =>
+  `
```

Comment:
nitpick but why does the error message start with a newline?

Reply:
Just looked a bit better to me in the GHA logs 🤷